Code/Tools/AzTestRunner/src/main.cpp

```diff
@@ -129,6 +129,13 @@ namespace AzTestRunner
             std::cout << "LIB: " << lib << std::endl;
         }
 
+        // Construct a retry command if test fails
+        std::string retry_command = "Retry command: " + std::string(argv[0]) + " " + lib + " " + symbol;
+        for (int i = 1; i < argc; i++)
```

Comment:
Still appears to not include the path to AzTestRunner, so I don't consider this command 'copy and paste ready'
Users aren't going to be in the <build>/bin/<config> folder, or necessarily even know what AzTestRunner is.

Reply:
argv[0] is the AzTestRunner executable